tests/markdown_empty/not_empty.md

```diff
@@ -0,0 +1 @@
+** Not empty **
```

Comment:
This one isn't necessary. All of the other .md files are already examples of `not_empty.md`

You could also move `empty.md` to `tests/markdown/` so you don't have a single file in the folder, but it's fine either way.